functions/venue.js

```diff
@@ -180,117 +159,6 @@ const checkIfUserHasVoted = async (venueId, pollId, userId) => {
-  if (!VALID_CREATE_TEMPLATES.includes(data.template)) {
-    throw new HttpsError(
-      "invalid-argument",
-      `Template ${data.template} unknown`
-    );
-  }
-
-  let owners = [context.auth.token.user_id];
-  if (data.owners) {
-    owners = [...owners, ...data.owners];
-  }
-
-  const venueData = {
-    template: data.template,
-    name: data.name,
-    config: {
-      landingPageConfig: {
-        checkList: [],
-        coverImageUrl: data.bannerImageUrl,
-        subtitle: data.subtitle,
-        description: data.description,
-      },
-    },
-    presentation: [],
-    quotations: [],
-    theme: {
-      primaryColor: data.primaryColor || DEFAULT_PRIMARY_COLOR,
-    },
-    host: {
-      icon: data.logoImageUrl,
-    },
-    owners,
-    entrance: data.entrance || [],
-    placement: { ...data.placement, state: PlacementState.SelfPlaced },
-    showChat: true,
-    parentId: data.parentId,
-    userStatuses: data.userStatuses || [],
-    showRadio: data.showRadio || false,
-    showUserStatus:
-      typeof data.showUserStatus === "boolean" ? data.showUserStatus : true,
-    radioStations: data.radioStations ? [data.radioStations] : [],
-    createdAt: Date.now(),
-    updatedAt: Date.now(),
-    hasSocialLoginEnabled: data.hasSocialLoginEnabled || false,
-  };
-
-  if (data.mapBackgroundImageUrl) {
-    venueData.mapBackgroundImageUrl = data.mapBackgroundImageUrl;
-  }
-
-  if (data.template === VenueTemplate.jazzbar) {
-    venueData.enableJukebox =
-      typeof data.enableJukebox === "boolean"
-        ? data.enableJukebox
-        : DEFAULT_ENABLE_JUKEBOX;
-  }
-  // @debt showReactions and showShoutouts should be toggleable for anything in HAS_REACTIONS_TEMPLATES
-  if (HAS_REACTIONS_TEMPLATES.includes(data.template)) {
-    venueData.showReactions =
-      typeof data.showReactions === "boolean"
-        ? data.showReactions
-        : DEFAULT_SHOW_REACTIONS;
-
-    venueData.showShoutouts =
-      typeof data.showShoutouts === "boolean"
-        ? data.showShoutouts
-        : DEFAULT_SHOW_SHOUTOUTS;
-
-    if (data.auditoriumColumns) {
-      venueData.auditoriumColumns = data.auditoriumColumns;
-    }
-
-    if (data.auditoriumRows) {
-      venueData.auditoriumRows = data.auditoriumRows;
-    }
-  }
-
-  if (IFRAME_TEMPLATES.includes(data.template)) {
-    venueData.iframeUrl = data.iframeUrl;
-    venueData.autoPlay = data.autoPlay;
-  }
-
-  if (ZOOM_URL_TEMPLATES.includes(data.template)) {
-    venueData.zoomUrl = data.zoomUrl;
-  }
-
-  switch (data.template) {
-    case VenueTemplate.animatemap:
-    case VenueTemplate.partymap:
-    case VenueTemplate.themecamp:
-      venueData.rooms = data.rooms;
-      venueData.roomVisibility = data.roomVisibility;
-      venueData.showGrid = data.showGrid ? data.showGrid : false;
-      break;
-
-    case VenueTemplate.playa:
-      venueData.roomVisibility = data.roomVisibility;
-      break;
-
-    default:
-      break;
-  }
-
-  return venueData;
-};
-
```

Comment:
Removed this function as adding support for a proper venue ID to be passed in didn't make any sense.